docs/contributing/ide_design_process.md

```diff
@@ -0,0 +1,20 @@
+# IDE Team Design Process
+
+## Discussions / Issues
+
+The goal is that folks can create and discuss feature suggestions without needing to have everything fully figured out. Feel free to file a new feature suggestion and members of the Roslyn-IDE team will add the "[Feature Request](https://github.com/dotnet/roslyn/issues?=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Feature+Request%22)" label. However, these feature suggestions won't be discussed until we have a proposal written for them.
+
+## Proposals
+
+### Adding a proposal
+
+For a feature proposal to be reviewed, a design proposal needs be appended to the GitHub issue. It should have the following information:
+
+- **Summary**: Brief summary of what this proposal is about.
+- **Background and Motivation**: What is the problem we are solving and in what context did you encounter it?
+- **Proposed Feature**: How the proposed solution solves the problem?
+- **Alternative Designs**: Were there other options you considered? How does this compare to features in other editors?
+
+### Getting the issue championed by the IDE team
+
+A member of [Roslyn-IDE](https://github.com/orgs/dotnet/teams/roslyn-ide) will need to decide to "champion" that feature proposal and add it to the "On Deck" tab in the [IDE Design Review](https://github.com/dotnet/roslyn/projects/40) project. Behind the scenes the IDE Team tries to meet once a week to discuss these issues.
```

Comment:
I wouldn't use `tries` here. I would just state that we meet once a week, and use `generally` as a qualifier if you think one is necessary. There are exceptions, of course, but that is our general rule.